devicon.json

```diff
@@ -181,6 +181,30 @@
         "color": "#4285F4",
         "aliases": []
     },
+    {
+        "name": "angular",
+        "altnames": [],
+        "tags": [
+            "framework"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain"
+            ],
+            "font": [
+                "original",
+                "plain"
+            ]
+        },
+        "color": "#FFFFFF",
+        "aliases": [
+            {
+                "base": "original",
+                "alias": "plain"
+            }
+        ]
```

Comment:
Hello @gabrielfins, if you include a specific plain svg, then you don't need to declare an alias between the original and the plain font.